torchci/rockset/commons/__sql/recent_pr_workflows_query.sql

```diff
@@ -21,6 +21,7 @@ WITH recent_shas AS (
 )
 SELECT
   w.id AS workflowId,
+  w.workflow_id as workflowUniqueId,
```

Comment:
I think having some explanation here is needed as there are yet another ID here to complicate things.  Here is what I get so far:

1. For job, `workflowId` is the workflow ID and `id` is the job ID
2. For workflow, `workflowId` is null, and `id` is now the workflow ID

A bit complicated, but that's ok.  Now there is `workflowUniqueId` from `w.workflow_id` :)

Reply:
I put it in the types file, does it also need to be in the sql file?

Reply:
Ah ok, that will do IMO